package.json

```diff
@@ -29,13 +29,14 @@
   "dependencies": {
     "@iarna/word-count": "^1.1.2",
     "@manuscripts/assets": "^0.6.4",
-    "@manuscripts/json-schema": "^2.2.2",
-    "@manuscripts/library": "1.3.2",
-    "@manuscripts/style-guide": "1.7.0",
+    "@manuscripts/json-schema": "2.2.2-LEAN-3093",
+    "@manuscripts/library": "1.3.2-LEAN-3092-0",
+    "@manuscripts/style-guide": "1.7.0-LEAN-3092-0",
     "@manuscripts/track-changes-plugin": "^1.7.5",
-    "@manuscripts/transform": "2.0.4",
+    "@manuscripts/transform": "2.0.4-LEAN-3093",
     "astrocite-eutils": "^0.16.4",
     "axios": "^1.2.2",
+    "mathjax-full": "^3.2.2",
```

Comment:
Let's use "mathjax" instead of "mathjax-full". Not sure why we were using the "full" version.